tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -766,6 +764,9 @@ TEST (nnstreamer_nnfw_mlapi, invoke_pipeline_00)
   status = ml_pipeline_src_get_handle (handle, "appsrc", &src_handle);
   EXPECT_EQ (status, ML_ERROR_NONE);
   /* register call back function when new data is arrived on sink pad */
+  sink_called_cnt = (guint *) g_malloc0 (sizeof (guint));
+  EXPECT_TRUE (sink_called_cnt != NULL);
```

Comment:
Refer: https://developer.gnome.org/glib/stable/glib-Memory-Allocation.html

You don't need to check the return of g_malloc0. The program terminates if it fails.
If SVACE gives you errors for not checking the return values of g_malloc0, make it false-positive and don't fix it.

Reply:
Oh.. Thank you! 
Updated.